frontend/components/core/ReTableInfo.vue

```diff
@@ -47,150 +47,160 @@
+              />
+              {{ group }}
+            </p>
+          </span>
+          <ul>
+            <li v-for="item in filteredResultsByGroup(group)" :key="item.id">
+              <div class="table-info__item">
+                <!-- <ReCheckbox v-if="globalActions" v-model="item.selectedRecord" class="list__item__checkbox" :value="item.name" @change="onCheckboxChanged($event, item.id, key)" /> -->
+                <span
+                  v-for="(column, idx) in columns"
+                  :key="idx"
+                  class="table-info__item__col"
+                >
+                  <span :class="column.class">
+                    <span v-if="column.type === 'action'">
+                      <a href="#"
+                        @click.prevent="onActionClicked(item.kind, item)"
+                        >{{ itemValue(item, column) }}
+                      </a>
+                    </span>
+                    <span v-else-if="column.type === 'link'">
+                      <NuxtLink v-if="item.link" :to="item.link"
+                        >{{ itemValue(item, column) }}
+                      </NuxtLink>
+                      <span v-else>{{ itemValue(item, column) }}</span>
+                      <re-action-tooltip tooltip="Copied">
+                        <ReButton
+                          title="Copy to clipboard"
+                          class="table-info__actions__button button-icon"
+                          @click.prevent="onActionClicked('copy-name', item)"
+                        >
+                          <svgicon name="copy" width="12" height="13" />
+                        </ReButton>
+                      </re-action-tooltip>
+                    </span>
+                    <ReDate
+                      v-else-if="column.type === 'date'"
+                      :date="itemValue(item, column)"
+                    />
                     <span
-                      v-if="itemValue(item, column) === 'Text2Text'"
-                      class="table-info__tag"
-                      >Experimental</span
+                      v-else-if="column.type === 'number'"
                     >
+                    {{itemValue(item, column) | formatNumber}}
+                    </span>
+                    <span v-else-if="column.type === 'object'">
+                      <p
+                        v-for="key in Object.keys(itemValue(item, column))"
+                        :key="key"
+                      >
+                        <strong>{{ key }}:</strong>
+                        {{ itemValue(item, column)[key] }}
+                      </p>
+                    </span>
+                    <span v-else-if="column.type === 'task'">
+                      {{ itemValue(item, column) }}
+                      <span
+                        v-if="itemValue(item, column) === 'Text2Text'"
+                        class="table-info__tag"
+                        >Experimental</span
+                      >
```

Comment:
same as before: a core component cannot include references to task or another app logic